assets/templates/dataset-filter/range-selector.tmpl

```diff
@@ -35,6 +35,67 @@
                         		<input class="btn btn--link underline-link" type="submit" value="Remove all" name="remove-all-range" />
 							  </div>
                               <div class="clearfix">
+                                 {{if eq $.Data.Title "Time"}}
+                                 <div class="margin-bottom--1">
+                                    <span class="font-size-17"><strong>Start Date</strong></span>
+                                 </div>
+                                 <div class="col col--md-12 col--lg-12 margin-bottom--2">
+                                    <label class="block margin-bottom--1" for="Month">
+                                    <strong>Month</strong>
+                                    </label>
+                                    <div class="select-alt">
+                                       <select class="select width-sm--10 width-md--10 width-lg--10" name="start-month" id="start-month">
+                                          {{ range $.Data.DateRangeData.MonthValues }}
+                                          <option value="{{.}}">{{.}}</option>
```

Comment:
Again i would indent here as you are looping over month values; easier to recognise loop.